src/dotnet/APIView/APIView/Model/TokenTreeModel.cs

```diff
@@ -0,0 +1,182 @@
+using System.Collections.Generic;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+
+namespace csharp_api_parser.TreeToken
+{
+    public enum StructuredTokenKind
+    {
+        Content = 0,
+        LineBreak = 1,
+        NoneBreakingSpace = 2,
+        TabSpace = 3,
+        ParameterSeparator = 4,
+        Url = 5
+    }
+
+
+    public class StructuredToken
```

Comment:
Can you add a comment about this class. For e.g. when is it used to make it easier for anyone who reads the code in the future?